src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -427,7 +428,14 @@ export class SimpleFilter extends Component {
     this.resetSimpleFilter();
   }
 
-  private handleGroupBy(data: IQuerySuccessEventArgs) {
+  private handleNoResults() {
+    this.findOrCreateWrapper().addClass('coveo-no-results');
+  }
+
+  private handleQuerySuccess(data: IQuerySuccessEventArgs) {
+    if (data.results.results.length > 0) {
```

Comment:
Is the `if` condition necessary here? This tells me when there are no results, both `querySuccess` and `noResults` are triggered. If that is the case, then why not add the `coveo-no-results` class in the `else` for this `if` instead of adding the listener on line 161 and the method `handleNoResults()`?